rfcs/0017-long-fork-consensus.md

```diff
@@ -0,0 +1,123 @@
+## Summary
+[summary]: #summary
+
+We currently do not support long forks. This means that to join the network you need a recent checkpoint, in other words to have always been online or to trust another party that has always been online.
+
+This adds support for a client to join the network at any point of time, regardless of if its been online recently. It adds this safely under a [50% + epsilon] security assumption. It also includes a mechanism to restore a chain to strength if this assumption was to be temporarily violated.
```

Comment:
I'm sure it's just a parameter I've forgotten from the paper, but how is epsilon defined here? Would be nice to just have a short mention here within the RFC.

Reply:
It just means >1/2. "honest majority". so 50% plus the smallest unit of currency.